sup.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from brutal.core.plugin import cmd
+from datetime import datetime
+import admanbot.plugins.calendar_sk as calendar_sk
```

Comment:
@Adman the real question is do we really want this thing to be a submodule?

If that's the case, you'll need to create a `sup/` directory, rename this file to `__init__.py` in that directory and put `calendar_sk.py` there too.